app/routes/authentication.py

```diff
@@ -123,6 +123,34 @@ def change_password():
     return "", 200
 
 
+@auth_blueprint.route("/change_email", methods=["POST"])
```

Comment:
In keeping with RESTFUL practices, this endpoint should be a PATCH / PUT on `/user/1` unless we are treating "email" as a separate resource.

Reply:
I agree Path user/<id> and then convert the the request json to json-schema